bazaar/interface.rst

```diff
@@ -1,12 +1,207 @@
 
-Bazaar Interfaces
-*****************
+Edustore Distributed API
+=========================
 
-LMS
-===
+draft v0.1
 
+.. contents::
 
-CMS
-===
 
+Signing and Authentication 
+==========================
 
+There are two supported methods:
+
+1. Header HMAC256 AUTH token in headers and JSON body (Similar: http://docs.aws.amazon.com/AmazonS3/latest/dev/RESTAuthentication.html)
+2. GET/POST request with params=<json>&authentication=<client_id:hash>
+
+1 Adding material
+==============================
+
+
+1.1 Searching and adding material to the LMS
+--------------------------------------------
+
+Learning material can be added with three use methods:
+
+1. User open Edustore repository directly.
+2. | User select materials from the Edustore repository to the LMS by using: 
+   | - Drag and Drop
```

Comment:
The term "drag and drop" is not defined anywhere in this document. What does it mean, where it is used and who is using it? I assume it means in the web page and by the end user, and that they are dragging a link on top of some element in some page somewhere. This needs more info how that should be implemented. Also, "Window to window" is not clear what it means related to "drag and drop".